packages/studio-base/src/panels/ThreeDeeRender/renderables/CoreSettings.ts

```diff
@@ -30,6 +30,18 @@ export const DEFAULT_PUBLISH_SETTINGS: RendererConfig["publish"] = {
   poseEstimateThetaDeviation: round(Math.PI / 12, 8),
 };
 
+const TopicsFilterOptions = [
+  { label: "All", value: "list-all" },
+  { label: "Visible", value: "list-visible" },
+  { label: "Not Visible", value: "list-not-visible" },
+];
+export const TopicsFilterSelect = {
+  label: "Filter Topics",
+  help: "Filter Topics By Visibility",
```

Comment:
I don't think we're consistent everywhere but I'm pretty sure we decided that we want to use sentence case
```suggestion
  label: "Filter topics",
  help: "Filter topics by visibility",
```